solvebio/resource/datasetmigration.py

```diff
@@ -26,6 +27,19 @@ class DatasetMigration(CreateableAPIResource, ListableAPIResource,
         ('updated_at', 'Updated'),
     )
 
+    @property
+    def source(self):
+        if not self.source_id:
```

Comment:
Didn't realize this was possible

Reply:
should it raise an exception? return None?

Reply:
ya....you are correct...not sure why i did that. serializers have required=True. removing